.circleci/config.yml

```diff
@@ -121,50 +134,95 @@ workflows:
   build_and_deploy:
     jobs:
       - build:
+          filters:
+            branches:
+              ignore:
+                - master
```

Comment:
We are going from building all branches to building all branches except for `master`, `support*`, `feature/*-i18n`, and `release*` branches. What is the reason here? Don't we still want to show `dev/<branch>` assets?

Reply:
I see, it would be an either-or situation with build and build_i18n

Reply:
This looks like it happens a lot. You don't want to have two different workflows?

Reply:
Yep! The build_i18n takes much longer and takes up more space in S3, so we decided to run the full build only for specific branches and tags.

Reply:
I think putting the i18n builds into a  build_and_deploy_i18n workflow would work well